get-testnet-tokens.md

```diff
@@ -122,7 +120,7 @@ aecli account balance ./my-ae-wallet -u https://sdk-testnet.aepps.com
 and we have it:
 ```
 Balance_________________________________ 5000000000000000000
-ID______________________________________ ak_maudCMZW7WyPrHFhpJNzjBtNBw62L7jceMyqkXkYVXtVfcDK1
+ID______________________________________ ak_hipJRoK6hfiRgcejJchpXwhDJKcN5AQ3yGmqmCU8rWJ2A3BXF
```

Comment:
I don't think that the changes to the addresses and transaction ID are needed. But it's cool

Reply:
I needed to test the entire tutorial to make sure each step was up to date, hence I had to use a fresh new address/new transaction.